includes/discovery/sensors/bitrate/terra-sdi480.inc.php

```diff
@@ -30,72 +30,70 @@
 $lowwarnlimit = 1 * 1000 * 1000; // 1 mbps
 $lowlimit = 100 * 1000; // 0.1 mbps
 
-if (is_array($pre_cache['sdi480status'])) {
+$oids = SnmpQuery::cache()->hideMib()->walk('TERRA-sdi480-MIB::sdi480status')->table(1);
```

Comment:
cache is also not appropriate for this os's use.

Cache only helps when the same OID is fetched with the same options in more than one place.